src/WireMock.Net/Settings/WireMockServerSettingsParser.cs

```diff
@@ -59,17 +63,35 @@ public static bool TryParseArguments(string[] args, [NotNullWhen(true)] out Wire
             WatchStaticMappings = parser.GetBoolValue("WatchStaticMappings"),
             WatchStaticMappingsInSubdirectories = parser.GetBoolValue("WatchStaticMappingsInSubdirectories"),
             HostingScheme = parser.GetEnumValue<HostingScheme>(nameof(WireMockServerSettings.HostingScheme)),
-            DoNotSaveDynamicResponseInLogEntry = parser.GetBoolValue(nameof(WireMockServerSettings.DoNotSaveDynamicResponseInLogEntry)),
-            QueryParameterMultipleValueSupport = parser.GetEnumValue<QueryParameterMultipleValueSupport>(nameof(WireMockServerSettings.QueryParameterMultipleValueSupport)),
-            Culture = parser.GetValue(nameof(WireMockServerSettings.Culture), strings => CultureInfoUtils.Parse(strings.FirstOrDefault()), CultureInfo.CurrentCulture)
+            DoNotSaveDynamicResponseInLogEntry =
+                parser.GetBoolValue(nameof(WireMockServerSettings.DoNotSaveDynamicResponseInLogEntry)),
+            QueryParameterMultipleValueSupport =
+                parser.GetEnumValue<QueryParameterMultipleValueSupport>(nameof(WireMockServerSettings
+                    .QueryParameterMultipleValueSupport)),
+            Culture = parser.GetValue(nameof(WireMockServerSettings.Culture),
+                strings => CultureInfoUtils.Parse(strings.FirstOrDefault()), CultureInfo.CurrentCulture)
         };
 
 #if USE_ASPNETCORE
-        settings.CorsPolicyOptions = parser.GetEnumValue(nameof(WireMockServerSettings.CorsPolicyOptions), CorsPolicyOptions.None);
-        settings.ClientCertificateMode = parser.GetEnumValue(nameof(WireMockServerSettings.ClientCertificateMode), ClientCertificateMode.NoCertificate);
-        settings.AcceptAnyClientCertificate = parser.GetBoolValue(nameof(WireMockServerSettings.AcceptAnyClientCertificate));
+        settings.CorsPolicyOptions =
+            parser.GetEnumValue(nameof(WireMockServerSettings.CorsPolicyOptions), CorsPolicyOptions.None);
+        settings.ClientCertificateMode = parser.GetEnumValue(nameof(WireMockServerSettings.ClientCertificateMode),
+            ClientCertificateMode.NoCertificate);
+        settings.AcceptAnyClientCertificate =
+            parser.GetBoolValue(nameof(WireMockServerSettings.AcceptAnyClientCertificate));
 #endif
 
+        TryParseLoggerSettings(settings, logger, parser);

```

Comment:
No need to add the `Try` prefix, because these new methods do not return a bool and the return result is not used anywhere.

Reply:
Noted, updated to match the above.